### PR TITLE
New command to list tests unit available

### PR DESCRIPTION
**Pull request by enzolutions**

### CODE REVIEW

File: src/Command/TestDebugCommand.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Component\Serialization\Yaml;
+use Drupal\simpletest\TestDiscovery;
+
+class TestDebugCommand extends ContainerAwareCommand
+{
+
+  protected function configure()
```
Comment (dmouse) [inline src/Command/TestDebugCommand.php]: pls, include the docblock @{inheritdoc}

File: src/Command/TestDebugCommand.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      ->addArgument('test-id', InputArgument::OPTIONAL,
+            $this->trans('commands.test.debug.arguments.resource-id'))
+      ->addOption('group', '', InputOption::VALUE_OPTIONAL,
+            $this->trans('commands.test.debug.options.group'))
+    ;
+
+    $this->addDependency('simpletest');
+  }
+
+  protected function execute(InputInterface $input, OutputInterface $output)
```
Comment (dmouse) [inline src/Command/TestDebugCommand.php]: please, include the docblock @{inheritdoc}

File: src/Command/TestDebugCommand.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  /**
+   * @param $output         OutputInterface
+   * @param $table          TableHelper
+   * @param $config_name    String
+   */
+  private function getTestByID($output, $table, $test_id)
+  {
+      $testing_groups = simpletest_test_get_all();
```
Comment (dmouse) [inline src/Command/TestDebugCommand.php]: I think that is better use the service container to get all test availables in Drupal:
container-get(service('test_discovery')->getTestClasses(null)

**Comment by enzolutions** [comment]
Changes requested applied

**Pull request merged**